controller-server/src/main/java/com/yahoo/vespa/hosted/controller/maintenance/MetricsReporter.java

```diff
@@ -71,6 +73,24 @@ public void maintain() {
         reportRemainingRotations();
         reportQueuedNameServiceRequests();
         reportInfrastructureUpgradeMetrics();
+        reportAuditLog();
+    }
+
+    private void reportAuditLog() {
+        final String OPERATOR = "operator";
```

Comment:
Variable should be lowercase, and `final` can be removed (we typically don't use final for local variables).